README.md

```diff
@@ -6,39 +6,39 @@
 
 ![Build](https://github.com/google/flax/workflows/Build/badge.svg?branch=main) [![coverage](https://badgen.net/codecov/c/gh/google/flax)](https://codecov.io/gh/google/flax)
 
-
 [**Overview**](#overview)
 | [**Quick install**](#quick-install)
 | [**What does Flax look like?**](#what-does-flax-look-like)
 | [**Documentation**](https://flax.readthedocs.io/)
 
-**📣 NEW**: Check out the [**NNX**](https://flax.readthedocs.io/en/latest/nnx/index.html) API!
+Released in 2024, Flax NNX is a new simplified Flax API that is designed to make
+it easier to create, inspect, debug, and analyze neural networks in
+[JAX](https://jax.readthedocs.io/). It achieves this by adding first class support
+for Python reference semantics. This allows users to express their models using
+regular Python objects, which are modeled as PyGraphs (instead of pytrees), enabling
```

Comment:
Lets remove this `PyGraph` mention as we don't use the term a lot.
```suggestion
regular Python objects, enabling
```

Reply:
Done!